lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -99,7 +100,34 @@ def scrub(transcript)
           gsub(/("number\\":\\")\d+/, '\1[FILTERED]').
           gsub(/("cvv\\":\\")\d+/, '\1[FILTERED]').
           gsub(/("cryptogram\\":\\")\w+/, '\1[FILTERED]').
-          gsub(/(source\\":\{.*\\"token\\":\\")\d+/, '\1[FILTERED]')
+          gsub(/(source\\":\{.*\\"token\\":\\")\d+/, '\1[FILTERED]').
+          gsub(/("token\\":\\")\w+/, '\1[FILTERED]')
+      end
+
+      def store(payment_method, options = {})
+        post = {}
+        MultiResponse.run do |r|
+          if payment_method.is_a?(NetworkTokenizationCreditCard)
+            response = r.process { verify(payment_method, options) }
```

Comment:
💬 Comment:

Maybe comment with an small explanation of this particular approach ?

Reply:
Because this comes from a Verify, I thought it could have two responses, the verify itself, and the "stored" (a formatted verify to look like a store)